Utils/azuretools-core/src/com/microsoft/azuretools/sdkmanage/IdentityAzureManager.java

```diff
@@ -150,6 +153,10 @@ public Subscription getSubscriptionById(String sid) {
             }
 
         } catch (Throwable e) {
+            if (StringUtils.isNotBlank(authMethodDetails.getClientId()) && authMethodDetails.getAuthType() == AuthType.SERVICE_PRINCIPAL) {
+                MvpUIHelperFactory.getInstance().getMvpUIHelper().forgetPasswordFromSecureStore(
+                    StringUtils.joinWith("|", "account", authMethodDetails.getClientId()));
```

Comment:
Shall we extract `StringUtils.joinWith("|", "account", authMethodDetails.getClientId())` to a method ?

Reply:
this code will be delete in future, handle it in future